SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: Artistic-1.0

Set some things that cannot be overriden in the perl Configure script to
generate with correct values for live-bootstrap.

NOTE: this patch CANNOT be applied to a non-live-bootstrap environment.

--- ../metaconfig-5.32.1~rc1/dist/U/archname.U	2022-02-26 10:51:45.343097807 +1100
+++ ../metaconfig-5.32.1~rc1/dist/U/archname.U	2022-02-26 10:51:51.742527859 +1100
@@ -72,5 +72,5 @@
 rp='What is your architecture name'
 . ./myread
 archname="$ans"
-myarchname="$tarch"
+myarchname="i386"
 
--- ../metaconfig-5.32.1~rc1/dist/U/Oldconfig.U	2022-02-27 10:55:04.890396204 +1100
+++ ../metaconfig-5.32.1~rc1/dist/U/Oldconfig.U	2022-02-27 11:00:31.324396204 +1100
@@ -109,16 +109,13 @@
 ?LINT:extern hostarch libswanted libs
 ?LINT:change hostarch libswanted libs
 : Determine the name of the machine
-myuname=`$uname -a 2>/dev/null`
-$test -z "$myuname" && myuname=`hostname 2>/dev/null`
 ?X: Special mention for Xenix, whose 'uname -a' gives us output like this:
 ?X:  sysname=XENIX
 ?X:  nodename=whatever
 ?X:  release=2.3.2 .. etc...
 ?X: Therefore, we strip all this variable assignment junk and remove all the
 ?X: new lines to keep the myuname variable sane... --RAM
-myuname=`echo $myuname | $sed -e 's/^[^=]*=//' -e 's/\///g' | \
-	./tr '[A-Z]' '[a-z]' | $tr $trnl ' '`
+myuname=""
 ?X: Save the value we just computed to reset myuname after we get done here.
 newmyuname="$myuname"
 has_uname=
@@ -277,7 +274,7 @@
 		;;
 	linux) osname=linux
 		case "$3" in
-		*)	osvers="$3" ;;
+		*)	osvers="4.14.341-openela" ;;
 		esac
 		;;
 	MiNT) osname=mint
